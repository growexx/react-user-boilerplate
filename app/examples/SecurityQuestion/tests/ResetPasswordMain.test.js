import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

// SampleComponent imports Link internally
// import SampleComponent from '../SampleComponent';
import ResetPasswordMain from '../ResetPassword/ResetPasswordMain';

describe('SampleComponent', () => {
  test('should render', () => {
    const component = renderer
      .create(
        <MemoryRouter>
          <ResetPasswordMain />
        </MemoryRouter>,
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
