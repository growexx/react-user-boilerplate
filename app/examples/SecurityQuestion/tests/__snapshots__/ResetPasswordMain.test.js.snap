// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleComponent should render 1`] = `
<div
  className="StyledResetPasswordMain__StyledResetPassword-a4cy1o-0 hDzXcA"
>
  <div
    className="sideContainer"
  >
    <div
      className="loginContainer"
    >
      <div
        className="registrationSideContainer"
      >
        <p
          className="title"
        >
          <span>
            New Here!
          </span>
        </p>
        <p
          className="subTitle"
        >
          <span>
            To join us, please sign up with your personal info
          </span>
        </p>
        <a
          href="/register"
          onClick={[Function]}
        >
          <button
            className="ant-btn"
            onClick={[Function]}
            type="button"
          >
            <span>
              SIGN UP
            </span>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="main"
  >
    <div>
      <p
        className="resetPassword"
      >
        Reset Password
      </p>
      <div
        className=""
      >
        <a
          className="link-reset"
          href="/forgot-password"
          onClick={[Function]}
        >
          Reset Via Link
        </a>
        <a
          className="link-security"
          href="/reset-password-security"
          onClick={[Function]}
        >
          Reset Via Security Question
        </a>
      </div>
    </div>
  </div>
</div>
`;
